src/hb-atomic.hh

```diff
@@ -80,27 +79,11 @@ _hb_atomic_ptr_impl_cmplexch (const void **P, const void *O_, const void *N)
 
 #include <atomic>
 
+#define STL_ATOMIC_IMPL
```

Comment:
Please prefix this with `HB_`. Otherwise looks good and I'll merge when back to computer.